lib/search-bar/defaultSuggestions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * If there is no suggestions matches this list will be displayed
+ */
+export default [{
+  boldTitle: 'Oh no! ',
```

Comment:
We need these strings to be translated to Danish ... 
@emiledskar can you help?